src/Resources/contao/library/Contao/Database/Installer.php

```diff
@@ -331,26 +324,15 @@ public function getFromDca()
 	 */
 	public function getFromFile()
 	{
+		/** @var KernelInterface $kernel */
 		global $kernel;
 
 		$table = '';
 		$return = array();
 
 		// Only check the active modules (see #4541)
-		foreach ($kernel->getContainer()->get('contao.resource_provider')->getResourcesPaths() as $strModule => $strFolder)
+		foreach ($kernel->getContainer()->get('contao.resource_locator')->locate('config/database.sql') as $strFile)
 		{
-			if (in_array($strModule, array('calendar', 'comments', 'faq', 'listing', 'news', 'newsletter')))
```

Comment:
You mean there could be a `system/modules/core` in a Contao 4 because of an update-leftover?
I think we should not handle that. Simply because the same could happen to Isotope, and there is no other correct solution than removing it manually.